client/src/components/NewFund.js

```diff
@@ -0,0 +1,233 @@
+    },
+    nested: {
+        paddingLeft: theme.spacing(4),
+    },
+}));
+
+var counter = 0;
+
+class NewFund extends Component {
+    state = {
+        open: false
+    }
+
+    // componentDidMount() {
+    //     this.state.user()
+    // }
+
+    MaterialUIPickers = () => {
+        // The first commit of Material-UI
+        const setSelectedDate = React.useState(new Date());
+      
+        const handleDateChange = date => {
+          setSelectedDate(date);
+        };
+    }
+
+    handleClick = () => {
+        counter++
+
+        if(counter % 2 === 1) {
+            this.setState({
+                open: true
+            })
+        } else {
+            this.setState({
+                open: false
+            })
+        }
+    };
+
+    multipleFileUploadHandler = (files) => {
+        const data = new FormData();    
+        let fileArray = Object.assign([], files)
+    
+        // If file selected
+        if ( files ) {
+            for ( let i = 0; i < fileArray.length; i++ ) {
+                data.append( 'imageGallery', fileArray[i], fileArray[i].name );
+            }
+            axios.post( 'http://localhost:3001/api/profile/multiple-file-upload', data, {
+                headers: {
+                    'accept': 'application/json',
+                    'Accept-Language': 'en-US,en;q=0.8',
+                    'Content-Type': `multipart/form-data; boundary=${data._boundary}`,
+                }
+            })
+            .then( ( response ) => {
+                if ( 200 === response.status ) {
+                    // If file size is larger than expected.
+                    if( response.data.error ) {
+                        if ( 'LIMIT_FILE_SIZE' === response.data.error.code ) {
+                            this.ocShowAlert( 'Max size: 2MB', 'red' );
+                        } else if ( 'LIMIT_UNEXPECTED_FILE' === response.data.error.code ){
+                            this.ocShowAlert( 'Max 4 images allowed', 'red' );
+                        } else {
+                            // If any error other than the exceeded file size                           
+                            this.ocShowAlert( response.data.error, 'red' );
+                        }
+                    } else {
+                        // Success
+                        let fileName = response.data;
+                        this.ocShowAlert( 'Files Uploaded', '#3089cf' );
+                    }
+                }
+            })
+            .catch( ( error ) => {
+                // If another error
+                this.ocShowAlert( error, 'red' );
+            });
+        } else {
+            // if file not selected throw error
+            this.ocShowAlert( 'Please upload file', 'red' );
+        }
+      };
+
+	// ShowAlert Function
+	ocShowAlert = ( message, background = '#3089cf' ) => {       
+		let alertContainer = document.querySelector( '#oc-alert-container' ),
+			alertEl = document.createElement( 'div' ),
```

Comment:
avoid using manipulating the DOM when using react. you can create a new component for this alert dialog